app/controllers/comments_controller.rb

```diff
@@ -1,8 +1,16 @@
 class CommentsController < ApplicationController
+
   before_action :set_comment, only: %i[ show edit update destroy ]
-  before_action :is_an_authorized_user, only: [:destroy, :create]
 
-  # make comments destroyable if you are not the currently signed in user
+  # before_action :is_an_authorized_user, only: [:destroy, :create]
+
+  before_action :ensure_current_user_is_author, only: [:destroy, :update, :edit]
+
+  before_action :ensure_user_is_authorized, only: [:show]
```

Comment:
these two are no longer needed, we are using Pundit to handle all authorizations